docs/docker.md

```diff
@@ -0,0 +1,6 @@
+# Create local Docker containers
+
+- `cd leaf`
+- `touch .env`
+- Follow the `docs/.env.example`, fill out the new `.env` file to match your local setup
```

Comment:
If we aren't using an `.env`, we can likely remove these lines. Alternatively, we could remove them and add relevant entries to a new `.env` with this PR.

Reply:
These docs were from the original docker/compose implementation, but later deleted. We should update the docs as needed, but I wanted to keep the initial PRs smaller in scope.